Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/Button.module.scss

```diff
@@ -125,3 +128,12 @@
     padding: 0;
   }
 }
+
+.lightSecondary {
+  // TODO: Fix this color
+  background-color: colors.$horizontal-line-gray;
```

Comment:
is this color fixed?

Reply:
I'll leave colors cleanup for a different PR as this one already contains too much cleanup code...